WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1190,6 +1220,10 @@ private extension NotificationsViewController {
         return AccountHelper.isDotcomAvailable() == false
     }
 
+    var shouldDisplaySettingsButton: Bool {
+        return AccountHelper.isDotcomAvailable() == true
```

Comment:
Can this one returns directly `AccountHelper.isDotcomAvailable()`?

Reply:
I know it's not related to this PR but I think also some line above you can do something like:
```
var shouldDisplayJetpackPrompt: Bool {
        return !AccountHelper.isDotcomAvailable()
    }
```

Reply:
Right, for the second item I think some folks do `== false` instead of `!` because it can be more obvious to read at a glance (it could be easy to not see the `!` at the start).